lms/templates/components/card/card.underscore

```diff
@@ -2,40 +2,40 @@
 <div class="wrapper-card-core">
     <div class="card-core">
         <% if (pennant) { %>
-            <small class="card-type"><%- pennant %></small>
+            <small class="card-type"><%= HtmlUtils.escape(pennant) %></small>
         <% } %>
         <h3 class="card-title"
             <% if (!_.isUndefined(srInfo)) { %>
                 aria-describedby="<%= srInfo.id %>"
             <% } %>
-            ><%- title %>
+            ><%= HtmlUtils.escape(title) %>
```

Comment:
@andy-armstrong The documented best practice doesn't have to exactly match the linter.  It would be possible to have the best practice be to use `<%= HtmlUtils.escape(...) %>`, but still have the linter accept usage of `<%- %>`.

I'm not sure where I stand on the best practice at this time.  Too bad we can't just override `<%-` to be the smarter one. :(

I know we need to make a decision quickly.